openshift.py

```diff
@@ -0,0 +1,750 @@
+import kubernetes
+import pprint
+import logging
+import requests
+import json
+import re
+from flask import Flask, redirect, url_for, request, Response
+
+import sys
+
+# application = Flask(__name__)
+
+
+class openshift:
+    headers = None
+    verify = False
+    url = None
+
+    def __init__(self, url, token, logger):
+        self.set_token(token)
```

Comment:
Opportunity to be more Pythonic: No need to have these be separate methods. Just set them here.